diff --git a/symbols/us b/symbols/us
index b3d4ef6c..00bc967d 100644
--- a/symbols/us
+++ b/symbols/us
@@ -1422,19 +1422,15 @@ xkb_symbols "hbs" {
   include "us"
   name[Group1]= "Serbo-Croatian (US)";
 
-  key <TLDE> { [ grave, asciitilde ] };
-  key <AE06> { [ 6, dead_caron, asciicircum, asciicircum ] };
-  key <AE08> { [ 8, asterisk, multiply, division ] };
-  key <AE11> { [ minus, underscore, endash, emdash ] };
-  key <AC09> { [ l, L, U01C9, U01C8 ] }; // ǉ ǈ
-  key <AB06> { [ n, N, U01CC, U01CB ] }; // ǌ ǋ
-  key <AB01> { [ z, Z, U01C6, U01C5 ] }; // ǆ ǅ
-  key <AD03> { [ e, E, EuroSign, cent ] };
-  key <AC03> { [ d, D, dstroke, Dstroke ] };
-  key <AC11> { [ dead_acute, quotedbl, apostrophe, apostrophe ] };
-  key <SPCE> { [ space, space, nobreakspace, nobreakspace ] };
-  key <AB08> { [ comma, less, U3003, guillemotright ] };
-  key <AB09> { [ period, greater, ellipsis, guillemotleft ] };
+  key <AD03> { [ e, E, EuroSign, cent ] }; // € ¢
+  key <AC02> { [ s, S, scaron, Scaron ] }; // š Š
+  key <AC03> { [ d, D, dstroke, Dstroke ] }; // đ Đ
+  key <AC09> { [ l, L, U1C9, U1C8 ] }; // lj Lj
+  key <AC10> { [ any, any, U1C6, U1C5 ] }; // dž Dž
+  key <AC11> { [ any,  any, cacute, Cacute  ] };  // ć Ć 
+  key <AB01> { [ z, Z, zcaron, Zcaron ] }; // ž Ž
+  key <AB03> { [ c, C, ccaron, Ccaron  ] }; // č Č
+  key <AB06> { [ n, N, U1CC, U1CB ] }; // nj  Nj
 
   include "level3(ralt_switch)"
 };
